packages/grid/_modules_/grid/models/gridSlotsComponent.ts

```diff
@@ -56,19 +53,19 @@ export interface GridSlotsComponent extends GridIconSlotsComponent {
   /**
    * Toolbar component rendered inside the Header component.
    */
-  Toolbar?: React.ElementType<GridToolbarContainerProps>;
```

Comment:
This is wrong. The props of a custom component are not the ones of its default container.

Reply:
It does look wrong indeed